projects/hyperium/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/hyperium/"
+main_repo: "https://github.com/hyperium/"
+primary_contact: "david@adalogics.com"
```

Comment:
I think hyperium is a great choice! Thanks David.

Quick question -- would you remain primary contact or is it better to switch @seanmonstar?

Reply:
Let's make me auto_ccs and @seanmonstar primary! Should be fixed now.